style.css

```diff
@@ -0,0 +1,892 @@
+  font-size: 20px;
+}
+
+.intro-2 h1 {
+  color: #ec5242;
+  font-weight: 100;
+  font-size: 20px;
+  text-align: center;
+}
+
+.intro h2 {
+  font-weight: 1000;
+  font-size: 38px;
+  line-height: 9vh;
+  background-image: url(Images/new\ dog.jpg);
+  background-position: right;
+  -webkit-background-clip: text;
+  color: transparent;
+  padding-bottom: 20px;
+}
+
+.intro-2 h2 {
+  font-weight: 1000;
+  font-size: 38px;
+  color: #ec5242;
+  padding-bottom: 20px;
+  text-align: center;
+}
+
+.info {
+  color: #272a31;
+  font-size: 13px;
+  font-weight: 500;
+  padding: 10px;
+  border: 2px solid white;
+  align-items: center;
+  margin-right: 4vw;
+}
+
+.info-2 {
+  color: #272a31;
+  font-size: 13px;
+  font-weight: 500;
+  padding: 10px;
+  border: 2px solid white;
+  background: #fff;
+  align-items: center;
+  margin-right: 10px;
+  line-height: 18px;
+  text-align: center;
+}
+
+.contact {
+  font-size: 10px;
+  text-align: center;
+  width: 200px;
+  align-self: center;
+  margin-top: 20px;
+  margin-bottom: 30px;
+}
+
+.contact a {
+  color: #272a31;
+}
+
+.logosection {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  border: 1px solid #000;
+  border-right: 0;
+  border-left: 0;
+}
+
+.aboutlogo {
+  text-align: center;
+  background-color: #fff;
+}
+
+.aboutlogo h3 {
+  text-align: center;
+  font-size: 22px;
+  color: #272a31;
+  font-weight: 550;
+  padding-top: 25px;
+}
+
+.aboutlogo p {
+  color: #272a31;
+  font-size: 12px;
+  font-weight: 500;
+  margin-right: 10px;
+  line-height: 18px;
+  width: 300px;
+  padding-left: 10px;
+}
+
+.theLogo {
+  margin-top: 60px;
+  line-height: 16px;
+  margin-bottom: 40px;
+  border: 1px solid #000;
+  padding-top: 40px;
+  padding-bottom: 40px;
+  margin-left: 15px;
+  margin-right: 15px;
+}
+
+.mlogo {
+  text-decoration: none;
+  font-size: 50px;
+  font-weight: 800;
+  color: #742820;
+  cursor: pointer;
+}
+
+.past {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 30px;
+}
+
+.allconcert {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  margin-bottom: 20px;
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.allconcert p {
+  color: #fff;
+  font-size: 15px;
+}
+
+.pastconcert {
+  display: flex;
+  background-color: #ae2e0b;
+  width: 288px;
+  height: 200px;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  color: white;
+  background-size: cover;
+  text-align: center;
+  margin-left: 5px;
+}
+
+.year {
+  position: absolute;
+  padding-bottom: 40px;
+  font-size: 20px;
+}
+
+.place {
+  position: absolute;
+}
+
+.concertimg {
+  width: 288px;
+  height: 200px;
+  filter: opacity(0.3);
+}
+
+.partner {
+  display: flex;
+  flex-direction: column;
+  background-color: #3c3e42;
+  align-items: center;
+}
+
+.partner h3 {
+  padding-top: 30px;
+  font-size: 25px;
+  color: #fff;
+}
+
+.logos {
+  display: flex;
+  list-style-type: none;
+  margin-left: 20px;
+  margin-right: 20px;
+}
```

Comment:
- [ ] Kindly add  ` align-items: center;` so as to match the given design for the logos section

Your design
![image](https://user-images.githubusercontent.com/81932478/205326661-246ca05c-5788-4825-b35c-e1f641181195.png)

Given design 
![image](https://user-images.githubusercontent.com/81932478/205326538-04d019a1-2f9c-4aa1-8545-940a26cd4bcb.png)



Your design after adding change
![image](https://user-images.githubusercontent.com/81932478/205326456-d3148193-7fd8-4f9c-bf9c-cac8b59685b7.png)